modules/tinymce/src/themes/silver/main/ts/Theme.ts

```diff
@@ -40,8 +40,8 @@ export default (): void => {
 
     // We wrap the `renderModeUI` function being returned by Render so that we can update
     // the getPopupSinkBounds mutable variable if required.
```

Comment:
Please add a comment here about how we can _never_ make this API `async`

Reply:
@TheSpyder  I added a comment here, but could it is better if the comment is in putted in `modules/tinymce/src/themes/silver/main/ts/modes/Iframe.ts`?

Reply:
no I think it belongs here, as anything you put in `Iframe` would also need to be added to `Inline`?